handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,89 @@
+|<img src="../../img/mdb-atlas.png" width="1500px"/>| Multi node with MongoDB Atlas | Here SystemLink is hosted on a dedicated server and connects to a MongoDB Atlas cluster. This is recommended for large scale deployments (>200 managed node, 10,000 test steps and results daily). MongoDB Atlas also simplifies database provisioning, operation, backup and restore. These features may be desireable at any scale of deployment. Refer to [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) for details on this service. |
+
+### Changing the index cache size for *single node* deployments
+
+To constrain the amount of memory used by MongoDB, the default installation of SystemLink caps the available memory for the database's index at 1GB. This can lead to high CPU usage if the index grows beyond 1GB and must be paged to disk. If you are using the Test Insights or Asset Management modules, NI recommends increasing the available memory for the index to 8GB. The following steps can be used to increase the available memory for the MongoDB index.
+  
+1. With a user with administrator privileges, log into the desktop of the SystemLink app server.
+1. Open **NI SystemLink Server Configuration**.
+1. Navigate to **NoSqlDatabase**.
+1. Under **The NoSqlServer is launched and managed by SystemLink**, adjust the value in **The cache size of the database in Gigabytes (GB)**.
+1. Click **Apply**.
+
+!!!note
+    Refer to [Sizing a SystemLink server](https://www.ni.com/documentation/en/systemlink/latest/setup/sizing-a-systemlink-server/) for guidance on the CPU, memory, and disk needed for SystemLink single node deployments.
+
+### Connecting to MongoDB in a Multi Node Configuration
+
+SystemLink supports connecting to standalone MongoDB instances and replica sets. While SystemLink can connect and use a sharded MongoDB cluster (`mongos`) it will not take full advantage of the capabilities enabled by sharded clusters.
+
+- NI does not support configurations where the MongoDB instance is used by multiple SystemLink servers.
+- NI does not support configurations where the MongoDB instance is also used by another production application.
+
+SystemLink makes available form input and connection strings to establish a connection between your SystemLink server and MongoDB. Form input may be used when connecting to a single standalone instance, and a connection string must be provided for replica sets and MongoDB Atlas.
+
+!!!note "Secure remote connections"
+    NI recommends securing communications between your SystemLink server and MongoDB instance with TLS. For details, refer to [TLS for Remote MongoDB Instances](/network-security/network-security/#tls-for-remote-mongodb-instances).
+
+!!!important
```

Comment:
It might be worth moving this from this collection of notes into the actual step where they enter the string. We have a lot of sundry notes in this section and people usually don't read all the 'front matter' to a series of steps.

Reply:
I've moved the TLS note to be inline with the workflow steps.